src/named_tuple.cr

```diff
@@ -168,6 +168,16 @@ struct NamedTuple
     hasher
   end
 
+  # Returns the types of this named tuple type.
+  #
+  # ```
+  # tuple_type = typeof({a: 1, b: "hello", c: 'x'})
+  # tuple_type.types # => {a: Int32, b: String, c: Char}
```

Comment:
Could we add another example accessing `types` from the instance?

```crystal
tuple = {a: 1, b: "hello", c: 'x'}
tuple.class.types # => {a: Int32, b: String, c: Char}
```

That will avoid some people trying to do `tuple.types` by mistake.

😁

Reply:
Sure. I will just change to your example. It's actually like the specs your suggestion.